src/nifc/codegen.nim

```diff
@@ -148,7 +148,10 @@ proc error(m: Module; msg: string; n: Cursor) {.noreturn.} =
   let info = n.info
   if info.isValid:
     let (file, line, col) = unpack(pool.man, info)
-    write stdout, pool.files[file]
+    if pool.files.hasId(file):
+      write stdout, pool.files[file]
+    else:
+      write stdout, "???"
```

Comment:
No, if the `info.isValid` it should be valid line information, the error happens earlier.